api/app/routes.py

```diff
@@ -1,6 +1,69 @@
-from app import app
+from . import app
+from . import models
 
+@app.route('/api/user', methods=['GET'])
+def getAllVoters():
+    return json.dump(User.query.all())
 
-@app.route('/')
-def index():
+@app.route('/api/user/<int:uuid>', methods=['GET'])
+def getVoterInfo(uuid):
+    return json.dump(User.query.filter_by(id=uuid).first())
+
```

Comment:
I think it's best you remove these changes to the routes from this PR; they seem to have some issues here and there (for instance, `json` isn't imported here). We can deal with them for the next PR.